jest.integration.js

```diff
@@ -11,11 +11,14 @@ const config = Object.assign({}, baseConfig, {
     '**/test/integration/spec/**/*.{js,ts}'
   ],
   setupFiles: [
+    'fake-indexeddb/auto',
     '<rootDir>/test/support/nodeExceptions.js',
     '<rootDir>/test/support/jest/jest.setup.js',
-    '<rootDir>/test/support/jest/jest.setup.node.js'
   ],
-  testEnvironment: 'node',
+  testEnvironment: 'jsdom',
+  moduleNameMapper: Object.assign({}, baseConfig.moduleNameMapper, {
+    '^./node$': './browser'
```

Comment:
Is this mapping intended to fix something in a dependency library?

Reply:
No, authjs has 2 different build paths for node and browsers. The difference in the builds utilizes swapping out modules between the target environments. This module mapping maps this environment to mimic the browser-build rather than the node-build

Reply:
I think it's out of scope. In addition, DPoP is only supported in browser environments

Reply:
Agreed it's not something needed for this PR, but do tests that cover functionality that works in non-browser environments run with this config?